.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -47,8 +47,12 @@ Enter your extended release note in the block below. If the PR requires addition
 
 <!--
 This section can be blank if this pull request does not require a release note.
-Please use the following format for linking documentation or pass the
-section below:
+
+If you decide to add links which point to resources within git repositories like
+KEPs or supporting documentation, please link to a specific commit and avoid
+linking directly to the master branch.
```

Comment:
Maybe add a reason: "These ensure that links point to a specific point in time, rather than a document that may change over time."

Reply:
Sure, added it as suggested. :+1: